content/source/docs/enterprise/api/account.md

```diff
@@ -10,6 +10,83 @@ sidebar_current: "docs-enterprise2-api-account"
 
 Account represents the current user interacting with Terraform.
 
+## Update your account info
+
+Your username and email address can be updated with this endpoint.
+
+`PATCH /account/update`
+
+Status  | Response                                        | Reason
+--------|-------------------------------------------------|----------
+[200][] | [JSON API document][] (`type: "users"`)         | Your info was successfully updated
+[401][] | [JSON API error object][]                       | Unauthorized
+[422][] | [JSON API error object][]                       | Validation errors
```

Comment:
In a (yet unmerged) PR, we went back and forth on how to describe 422 errors, and decided that "Malformed request body" was more specific and useful than "validation errors".